src/components/QrcodeStream.vue

```diff
@@ -6,279 +6,283 @@
     The first element is at the very bottom and subsequent elements are added on top.
     -->
     <video
-      ref="video"
+      ref="videoRef"
       :class="{ 'qrcode-stream-camera--hidden': !shouldScan }"
       class="qrcode-stream-camera"
       autoplay
       muted
       playsinline
-    ></video>
+    />
 
-    <canvas
-      ref="pauseFrame"
-      v-show="!shouldScan"
-      class="qrcode-stream-camera"
-    ></canvas>
+    <canvas ref="pauseFrameRef" v-show="!shouldScan" class="qrcode-stream-camera" />
 
-    <canvas ref="trackingLayer" class="qrcode-stream-overlay"></canvas>
+    <canvas ref="trackingLayerRef" class="qrcode-stream-overlay" />
 
     <div class="qrcode-stream-overlay">
-      <slot></slot>
+      <slot />
```

Comment:
For example this. Are these two definitely the same?

Reply:
Yes